web/src/agents/node/get_balance.rs

```diff
@@ -0,0 +1,41 @@
+use at2_node::client::{self, Client};
+use at2_ns::ThinUser;
+use wasm_bindgen_futures::spawn_local;
+use yew::worker::*;
+
+use crate::config::Config;
+
+/// Get the balance of an user
```

Comment:
```suggestion
/// Get the balance of a user
```

the binding `n` is not used here as it's pronounced `yuser`, and the `y` doesn't ask for a binding `n`...

https://english.stackexchange.com/questions/105116/is-it-a-user-or-an-user

Reply:
English is weird some times, thanks for the link :)

Reply:
s/some times/sometimes/ ;)